### ISSUE TITLE
SSH key not working with git clone

### ISSUE THREAD

**Issue by lhorace** [ansible-semaphore/semaphore#165]
Hello,

I am not too sure if I am completely understanding this correctly. I've created a Playbook Repository and I assigned it to use ssh key semaphore@ansible.lhprojects.int. Then I created a task template and selected semaphore@ansible.lhprojects.int key once again. However, when I run the task, it fails. See below.

```
Started: 9

access key semaphore@ansible.lhprojects.int installed
Cloning repository
Cloning into 'repository_1'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Failed updating repository: exit status 128
```

As far as I can tell, on the SSH server side, no key was sent. I am not sure if this is intended behaviour or do I need to actually set a private key in ~/.ssh/id_rsa for the user semaphore is running as?

On a unrelated issue and I am fairly new with Ansible. In my git repository, I've create all my playbooks in subfolders. IE:

```
Ansible_Playbooks/
├── README
└── update_all_systems
    ├── roles
    │   └── update
    │       └── tasks
    │           └── main.yml
    └── site.yml
```

What I am worried about, can I specify specific playbooks, in situations where there are more than 1 playbook? Additionally, what should I specify for Playbook Name? site.yml?

Regards

**Comment by andviro** [comment]
Apparently, semaphore uses Git 2.x feature -- 'GIT_SSH_COMMAND' environment variable to provide ssh identity. Try to upgrade your Git installation and re-run.

**Comment by lhorace** [comment]
Can this be added in Documentation somewhere?

**Comment by dsmorse** [comment]
I updated the installation page: https://github.com/ansible-semaphore/semaphore/wiki/Installation

**Comment by knsr** [comment]
Hi @dsmorse,
Think that the wiki needs to be updated, because the "GIT_SSH_COMMAND" env var is available on **git > 2.3 version only**. The only ref I found is this [GitHub blog post](https://github.com/blog/1957-git-2-3-has-been-released).

Tested today on Semaphore 2.1.0.

Thanks 😃
